website/content/docs/operations/stateful-workloads.mdx

```diff
@@ -0,0 +1,246 @@
+---
+layout: docs
+page_title: Considerations for Stateful Workloads
+description: |-
+  Learn about persistent storage options for stateful workloads on Nomad.
+---
+# Considerations for Stateful Workloads
+
+By default, HashiCorp Nomad's allocation storage is ephemeral. Nomad can discard
+it during new deployments, when rescheduling jobs, or if it loses a client. This
+is undesirable when running persistent workloads such as databases.
+
+This document explores the options for persistent storage of workloads running
+in Nomad. The information provided is for practitioners familiar with Nomad and
+with a foundational understanding of storage basics. 
+
+At the end of this article, we provide a [storage
+comparison](#storage-comparison) table with the advantages, disadvantages, and
+the ideal use-case for each storage option. This guidance will help you
+understand and choose the best storage option for your Nomad allocation storage.
+
+This document addresses traditional storage you can attach as a filesystem and
+does not cover object storage such as AWS S3 or MinIO. 
+
+## Considerations
+
+Consider access patterns, performance, reliability and availability needs, and
+maintenance to choose the most appropriate storage strategy. 
+
+Local storage is performant and available. If it has enough capacity it does not
+need much maintenance. But, it is not redundant; if a single node, disk, or
+group of disks fails, data loss and service interruption will occur.
+
+A geographically distributed networked storage with multiple redundancies,
+including disks, controllers, and network paths, provides higher availability
+and resilience, and can tolerate multiple hardware failures before risking data
+loss. But, it relies on network performance, can have higher latency, be
+lower-throughput, and need more maintenance.
+
+Consider whether Nomad is running in the public cloud or on-premises, and what
+storage options are available in that environment. From there, the most optimal
+choice will depend your organizational and application needs.
+
+### Public cloud
+
+Public cloud providers offer different storage services with various tradeoffs.
+Usually they're comprised of local disks (physically attached and very
+performant, but not very persistent), network attached block devices (performant
+and very persistent, but usually only attachable to one instance at a time), and
+networked shared storage (less performant, but can be accessed from multiple
+instances). 
```

Comment:
"very" is doing a lot of handwaving in this paragraph. I'd get rid of all the parentheticals and let the tables speak for themselves.